OxfordBodleian/Juel-Jensen/BDLaethe31.xml

```diff
@@ -37,9 +41,566 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msContents>
                         <summary/>
                         
-                        <!--<msItem xml:id="ms_i1">
-                      
-                        </msItem>-->
+                        <msItem xml:id="ms_i1">
+                      <locus from="1ra1" to="1ra20"/>
+                            <title type="complete" ref="LIT5920JohnBeg"/>
+                            <note>The text consists of John 1:1–5, followed an elaboration on how, similar to how the Darkness could not overcome the Light
+                                in the Gospel excerpt,
+                                demons will not prevail over the owner of the scroll.
+                            </note>
+                            <incipit xml:lang="gez">
+                                <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡ ቀዳሚ</hi>ሁ፡ ቃል፡ ውእቱ፡ ወውእቱ፡ 
+                                ቃል<supplied reason="omitted">፡</supplied> ኀበ፡ እግዚአብሔር፡ ውእቱ፡ ወእ<hi rend="rubric">ግዚአብሔር፡ ውእቱ፡ 
+                                    ቃል፡ ወከማሁ፡ ቀዳሚሁ፡ እምቀዲሙ<supplied reason="omitted">፡</supplied></hi> ኀበ፡ እግዚአብሔር፡ ውእቱ፡ ቃል፡ 
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                ከማሁ፡ ኢይቅረብዋ፡ መናፍስት<supplied reason="omitted">፡</supplied> ርኩሳን፡ ወሰብእ፡ መሠርያን፡ 
+                                ወአጋንንት፡ ፀዋጋን፡ ከመ፡ ኢይቅረቡ፡ ኀበ፡ ነፍሳ፡ ወሥጋሀ፡ ለአመትከ፡ 
+                                <choice>
+                                    <orig><hi rend="rubric">አመተ፡ ማርያም፡</hi></orig>
+                                    <corr>
+                                        ወለተ፡
+                                        <choice>
+                                            <orig>ሩፋል።።</orig>
```

Comment:
ሩፋኤል perhaps ኤ is missing?

Reply:
No, in this case it is really not there. I can add a sic